backend/handlers/institution_children_handler.py

```diff
@@ -0,0 +1,96 @@
+"""Institution Children handler."""
+
+from google.appengine.ext import ndb
+
+from utils import Utils
+from util import login_required, Notification, NotificationsQueueManager
+from utils import json_response
+from custom_exceptions import NotAuthorizedException
+from custom_exceptions import EntityException
+from send_email_hierarchy import RequestLinkEmailSender
+
+from models import Institution
+
+from service_messages import send_message_notification
+from service_entities import enqueue_task
+from util import get_subject
+
+from . import BaseHandler
+
+__all__ = ['InstitutionChildrenHandler']
+
+def create_notification(user, receiver_institution, sender_institution, create_message):
+    message = create_message(
+        user_key=user.key, 
+        current_institution_key=user.current_institution, 
+        receiver_institution_key=receiver_institution.key, 
+        sender_institution_key=sender_institution.key)
+
+    notification = Notification(
+        message=message,
+        entity_key=receiver_institution.key.urlsafe(),
+        notification_type='REMOVE_INSTITUTION_LINK',
+        receiver_key=receiver_institution.admin.urlsafe()
+    )
+
+    notification_id = NotificationsQueueManager.create_notification_task(notification)
+    return notification_id
+
+
+class InstitutionChildrenHandler(BaseHandler):
+    """Institution Children Handler."""
+
+    @json_response
+    @login_required
+    @ndb.transactional(xg=True)
+    def delete(self, user, institution_key, institution_link):
+        """
+        Handle delete children link between institutions.
+
+        This handler remove the children link between two institutions. 
+        If the parameter isParent is true, it means that the removal 
+        request has been made from a child institution, otherwise 
+        the request has been made by a parent institution.
+        """
+        # holds the reference of the child intitution.
```

Comment:
now, that the method is separated, we can rename the parameters to institution_children_key and institution_parent_key and remove this doc